packages/charts/react-charting/etc/react-charting.api.md

```diff
@@ -194,6 +194,7 @@ export interface IAccessibilityProps {
 
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33475&commitId=0cb14835e191964bc2ae2e90ff8bda4f570ff659&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Left center.chromium.png|2616| Changed|
|Callout.Bottom Right Edge RTL.chromium.png|1593| Changed|
|Callout.Top center.chromium.png|2178| Changed|
|Callout.Bottom auto edge.chromium.png|2309| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>